v4/export/config.go

```diff
@@ -61,6 +62,12 @@ func (conf *Config) getDSN(db string) string {
 	return fmt.Sprintf("%s:%s@tcp(%s:%d)/%s?charset=utf8mb4", conf.User, conf.Password, conf.Host, conf.Port, db)
 }
 
+type GlobalConfig struct {
+	EscapeBackslash bool
+}
+
+var globalConfig GlobalConfig
```

Comment:
No I mean passing the config into `ToString()`. Are you supposed to rely on a frequently changing global variable in DM?

Reply:
Do you mean to pass the config from `v4/export/dump.go:Dump` -> `dumpDatabases` -> ... -> `ToString()`?

Reply:
It should be passed into in the SQL implementation of `WriteInsert`, because whether to escape backslash is an SQL-output-only concern. (But for now the abstraction is totally missing :shrug:).

Reply:
Revised in https://github.com/pingcap/dumpling/pull/49/commits/4b1c80c39524bf501a5dcc7d3893f70109592500, PTAL